### ISSUE TITLE
Update to black v22.1.0

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#3851]
Black is finally out of beta :)

I have this working locally running ``black -t py37`` but blocked by https://github.com/asottile/blacken-docs/issues/143

Most visible change is no spaces on simple power expressions, see also: https://black.readthedocs.io/en/latest/change_log.html

**Comment by peterjc** [comment]
Issue was a duplicate of https://github.com/asottile/blacken-docs/issues/141 which is fixed, just waiting on a release with the fix.

**Comment by peterjc** [comment]
Fixed in blacken-docs v1.12.1
